chain-signatures/contract/src/lib.rs

```diff
@@ -677,12 +683,16 @@ impl VersionedMpcContract {
     }
 
     #[private]
-    pub fn sign_helper(&mut self, request: SignatureRequest) {
+    pub fn sign_helper(&mut self, contract_signature_request: ContractSignatureRequest) {
         match self {
             Self::V0(mpc_contract) => {
+                // refund must happen in clear_state_on_finish, because regardless of this success or fail
```

Comment:
Should we move this comment to where the refund is actually happening?

Reply:
OK